linux_os/guide/services/ssh/ssh_server/sshd_use_strong_kex/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Use Only Strong Key Exchange algorithms'
+
+description: |-
+    Limit the Key Exchange to strong algorithms.
+    The following line in <tt>/etc/ssh/sshd_config</tt> demonstrates use
+    of those:
+    <pre>KexAlgorithms {{{ sshd_strong_kex }}}</pre>
```

Comment:
I'm sorry for not being detailed yesterday. Using the variable will not work this way. You have to use the `xccdf_value` macro here and at other places in the rule. You will also have to modify OVAL, Ansible, and Bash to use the value. I see that the Bash and Ansible is already done here. I think that for the OVAL you won't be able to use the macro and you will have to write some OVAL code now. Take a look at other rules that are using a variable, eg. https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/selinux/selinux_state